### DISCUSSION TITLE
I'm going to do btech cse, for me any tips?

### DISCUSSION THREAD

**Discussion by josh-sidheswar** [pyg-team/pytorch_geometric#3498] [Q&A]
How to start free lancing and what to do now I need some suggestions so kindly if  someone
Help regards this, it will change my career direction

**Comment by rusty1s** [comment]
Any contributions are highly welcome, but we sadly cannot pay you as we are a non-profit org. If you tell me which parts you are interested in contributing, I can give you more detailed information :)
